### PR TITLE
Bump rancher-webhook to v0.2.0-beta01

### PR DESCRIPTION
**Pull request by thedadams**
Related issue:
https://github.com/rancher/rancher/issues/32388

### CODE REVIEW

**Review by cmurphy — APPROVED** [review]

**Review by jiaqiluo — DISMISSED** [review]

**Comment by thedadams** [comment]
Fixed conflict in `index.yaml`.

**Review by jiaqiluo — APPROVED** [review]

**Review by cmurphy — APPROVED** [review]

File: charts/rancher-webhook/rancher-webhook/1.0.0+up0.2.0-beta01/Chart.yaml [hunk 1]
```diff
@@ -5,11 +5,11 @@ annotations:
   catalog.cattle.io/os: linux
   catalog.cattle.io/release-name: rancher-webhook
 apiVersion: v2
-appVersion: 0.2.0-alpha04
+appVersion: 0.2.0-beta01
```
Comment (cbron) [inline charts/rancher-webhook/rancher-webhook/1.0.0+up0.2.0-beta01/Chart.yaml:8]: Are we moving from beta to a 0.2.0 before release ? Or sticking with beta in release ? Also is there a reason we aren't just releasing a 1.0.0 ?

Comment (thedadams) [reply charts/rancher-webhook/rancher-webhook/1.0.0+up0.2.0-beta01/Chart.yaml:8]: I'm not aware of the version decisions for the webhook.

@ryansann or @dramich: can either of you answer these questions?

Comment (ryansann) [reply charts/rancher-webhook/rancher-webhook/1.0.0+up0.2.0-beta01/Chart.yaml:8]: I'll defer to @dramich on this. But I do know we have been using the v0.1.x line for 2.5 charts and v0.2.x line for 2.6 charts

Comment (dramich) [reply charts/rancher-webhook/rancher-webhook/1.0.0+up0.2.0-beta01/Chart.yaml:8]: The goal is to release 0.2.0 before (or with) 2.6 releases, hence moving this to beta now. 0.1.0 is 2.5 and 0.2.0 is 2.6 line.

**Review by cbron — APPROVED** [review]

**Pull request merged**